tests/Endpoints/IndexTest.php

```diff
@@ -37,6 +37,10 @@ public function testIndexSettingsAlwaysReturnArrays(): void
         /* @phpstan-ignore-next-line */
         $this->assertIsArray($this->index->getDisplayedAttributes());
         /* @phpstan-ignore-next-line */
+        $this->assertIsArray($this->index->getFacetingAttributes());
+        /* @phpstan-ignore-next-line */
```

Comment:
Do you know why this `/* @phpstan-ignore-next-line */` is still required?

Reply:
When removing this line, I get the following PHPStan error:

```
Call to method PHPUnit\Framework\Assert::assertIsArray() with array will always evaluate to true.
```

That's because the return typehint of the method is `: array` ... which means this test doesn't make much sense (no need to test it as PHP does the check).

I updated the test and removed the PHPStan docblocks.